docs/content/menu/index.md

```diff
@@ -3,66 +3,65 @@ headless: true
 ---
```

Comment:
I've broken links but it's for the best. Now we have simpler structure:

``
content/
  docs/
  menu/
  posts/
```